src/contracts/mainnet/AaveV3EthAddCRVPoolPayload.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.17;
+
+import {AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+import {AaveV3Ethereum} from 'aave-address-book/AaveV3Ethereum.sol';
+import {IGenericV3ListingEngine, AaveV3ListingEthereum} from 'aave-helpers/v3-listing-engine/AaveV3ListingEthereum.sol';
+
+/**
+ * @title List CRV on AaveV3Ethereum
+ * @author Llama
+ * @dev This proposal lists CRV on Aave V3 Ethereum
+ * Governance: https://governance.aave.com/t/arfc-add-crv-to-ethereum-v3/11532
+ * Snapshot: https://snapshot.org/#/aave.eth/proposal/0xa0b5336692735fb4288537646aec5819cb4cbf01fc3a8a7cb06c9e62db708055
+ */
+contract AaveV3EthAddCRVPoolPayload is AaveV3ListingEthereum {
+  address public constant INTEREST_RATE_STRATEGY = 0x76884cAFeCf1f7d4146DA6C4053B18B76bf6ED14;
+
+  constructor() AaveV3ListingEthereum(IGenericV3ListingEngine(AaveV3Ethereum.LISTING_ENGINE)) {}
+
+  function getAllConfigs() public pure override returns (IGenericV3ListingEngine.Listing[] memory) {
+    IGenericV3ListingEngine.Listing[] memory listings = new IGenericV3ListingEngine.Listing[](1);
+
+    listings[0] = IGenericV3ListingEngine.Listing({
+      asset: AaveV2EthereumAssets.CRV_UNDERLYING,
+      assetSymbol: 'CRV',
+      priceFeed: AaveV2EthereumAssets.CRV_ORACLE,
```

Comment:
I know this won't change. But just double checking if you should be using the AaveV2EthereumAssets here

Reply:
It doesn't exist in V3 yet to add so I think it makes sense to use the library where we know the values are right.

Reply:
SGTM

Reply:
Leaving a question here for @sakulstra as he just pointed out on Telegram that the V2 Oracle is CRV/ETH and it being 10PM I'd just rather you see this tomorrow morning:

The AIP/spec didn't list the oracle so I wrongly assumed would be the same one as the V2 one. I now know I could have used the tenderly cli to see the UI and validate this but, if there's only an ETH based oracle, what other changes/set up needs to be done for this to work correctly on V3? 

Thank you.

Reply:
it's not really a v3 vs v2 think with the oracle.

Aave can have different base units, but they are for the whole pool - aave v2 ethereum afaik is the only pool using eth base, every other v2 and v3 pool uses USD.